src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsStep.java

```diff
@@ -45,6 +46,11 @@ public DescriptorImpl getDescriptor() {
 
         private List<Template> templates = Collections.emptyList();
 
+
```

Comment:
Sorry, my OCD is talking :) so I'd prefer to have only one blank line between the statements.

It seems that here are a total of two blank lines between the statements. Can you please remove one line? 

And add a new blank line after the end of `setTemplates` method, just before `public DescriptorImpl() {`?